src/material/tabs/paginated-tab-header.ts

```diff
@@ -138,10 +138,10 @@ export abstract class MatPaginatedTabHeader implements AfterContentChecked, Afte
   private _selectedIndex: number = 0;
 
   /** Event emitted when the option is selected. */
-  readonly selectFocusedIndex: EventEmitter<number> = new EventEmitter<number>();
+  @Output() readonly selectFocusedIndex = new EventEmitter<number>();
 
   /** Event emitted when a label is focused. */
-  readonly indexFocused: EventEmitter<number> = new EventEmitter<number>();
+  @Output() readonly indexFocused = new EventEmitter<number>();
```

Comment:
These aren't supposed to be outputs on the base class, because they're internal event emitters that are mostly used to keep the state in sync. We are using them as inputs on the `mat-tab-header` which is an internal component, but doing it on the base class will expose them on `mat-tab-nav-bar` which is public-facing. We're already doing the correct thing which is not to decorate them, but to have `outputs: ['selectFocusedIndex', 'indexFocused']` on the tab header.

Reply:
So, as you can see it the 
[print](https://user-images.githubusercontent.com/11965907/90589823-65b65600-e1b5-11ea-926f-33602ab033ed.png) I posted, it's already exposed in `mat-tab-nav-bar`, however it isn't decorated with the correspondent decorator due to the parent class not having its decorators. That being said, what's the proper way to fix it then?

Reply:
It's exposed as a property, but it's not an input. I don't think the docs are a good enough justification to mark something as an input.

Reply:
You're correct. Let me revert this.